tests/unit-core/test-backtrace.c

```diff
@@ -198,5 +198,46 @@ main (void)
 
   jerry_cleanup ();
 
+  jerry_init (JERRY_INIT_EMPTY);
```

Comment:
Would it be possible to move this new test to it's own function? That way it is clear that it is a single test case.